virttest/utils_test/libvirt.py

```diff
@@ -1240,6 +1240,7 @@ def ping_vm(self, vm, test, params, uri=None, ping_count=10,
             server_ip = params.get("server_ip")
             src_uri = "qemu:///system"
             vm.connect_uri = uri
+            vm_ip[vm.name] = vm.get_address()
```

Comment:
move vm_ip[vm.name] to ahead 1238

Reply:
After checked the code, I found we can't move vm_ip[vm.name] outside if-else, because in else, it needs to start vm if vm is not up, then get vm address